example/basic/basic_hcube.cxx

```diff
@@ -26,81 +26,137 @@
 #include <t8_forest.h>
 #include <t8_cmesh_vtk.h>
 
+/** Construct the cmesh given the dimension of the examples
+ * \param[in] dim           The dimension of the example
+ * \param[in] do_partition  Option to partition the cmesh  */
```

Comment:
Please specify the concrete behavior of the parameter.
"If true (non-zero) the cmesh will be partitioned across the processes..."